.github/libs/GithubUtils.js

```diff
@@ -189,6 +190,17 @@ class GithubUtils {
                 );
                 console.log('Filtering out the following automated pull requests:', automatedPRs);
 
+                const internalQAPRMap = _.reduce(
+                    _.filter(data, pr => !_.isEmpty(_.findWhere(pr.labels, {name: INTERNAL_QA_LABEL}))),
+                    (map, pr) => {
+                        // eslint-disable-next-line no-param-reassign
+                        map[pr.html_url] = _.pluck(pr.assignees, 'login');
```

Comment:
I noticed that if any of the objects in an array are missing the `pluck`'d key, it inserts `undefined` into the resultant array:

```node
> _.pluck([{a: 'b'}], 'a');
[ 'b' ]
> _.pluck([{a: 'b'}], 'blah');
[ undefined ]
> _.pluck([{a: 'b'}, {a: 'c'}], 'blah');
[ undefined, undefined ]
> _.compact(_.pluck([{a: 'b'}, {a: 'c'}], 'blah'));
[]
```

So I think we should avoid that and wrap this in a call to `_.compact`:

```suggestion
                        map[pr.html_url] = _.compact(_.pluck(pr.assignees, 'login'));
```